src/main/java/net/minecraftforge/fmp/capabilities/CapabilityWrapperItemHandler.java

```diff
@@ -0,0 +1,124 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.Collection;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.items.CapabilityItemHandler;
+import net.minecraftforge.items.IItemHandler;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+public class CapabilityWrapperItemHandler implements ICapabilityWrapper<IItemHandler>
+{
```

Comment:
We already have these classes, https://github.com/MinecraftForge/MinecraftForge/tree/master/src/main/java/net/minecraftforge/items/wrapper